cabal.project

```diff
@@ -7,14 +7,17 @@ packages:
   ouroboros-network/ouroboros-network-testing
   ouroboros-network/ouroboros-network
   ouroboros-network/ouroboros-consensus
-  ouroboros-network/ouroboros-consensus/ouroboros-consensus-mock
-  ouroboros-network/ouroboros-consensus/ouroboros-consensus-test-infra
-  ouroboros-network/ouroboros-consensus-cardano
   ouroboros-network/ouroboros-consensus-byron
   ouroboros-network/ouroboros-consensus-byron-test
   ouroboros-network/ouroboros-consensus-byronspec
+  ouroboros-network/ouroboros-consensus-cardano
+  ouroboros-network/ouroboros-consensus-cardano
+  ouroboros-network/ouroboros-consensus-cardano-test
+  ouroboros-network/ouroboros-consensus-mock
+  ouroboros-network/ouroboros-consensus-mock-test
   ouroboros-network/ouroboros-consensus-shelley
   ouroboros-network/ouroboros-consensus-shelley-test
+  ouroboros-network/ouroboros-consensus-test
   ouroboros-network/io-sim
   ouroboros-network/io-sim-classes
   ouroboros-network/ntp-client
```

Comment:
I don't really pay attention to this repo, but some remarks unrelated to this PR:

* Is `cardano-shell` still used?
* This file says `index-state: 2020-07-14T22:50:55Z`, while we use `index-state: 2020-07-15T00:00:00Z` in most repos. It's the same index state anyway (using the latter gives a warning about falling back to the former), but still a bit confusing.
* Is the `graphviz == 2999.20.0.3` needed?
* ```
  package quickcheck-state-machine
    tests: False
  ```
  is no longer needed

Reply:
I have no idea if `cardano-shell` is used, as well as if `graphivz` constraint is still needed.